pkgs/applications/editors/brackets/default.nix

```diff
@@ -9,55 +11,56 @@ let
     ];
   };
 in
-stdenv.mkDerivation rec {
-  name = "brackets-${version}";
-  version = "1.8";
-
-  src = fetchurl {
-    url = "https://github.com/adobe/brackets/releases/download/release-${version}/Brackets.Release.${version}.64-bit.deb";
-    sha256 = "0b2k0vv1qwmsg1wckp71yrb86zp8zisskmzzvx9ir19bma9jzr42";
-    name = "${name}.deb";
-  };
+  stdenv.mkDerivation rec {
```

Comment:
I usually just rebase onto master (or whatever branch I'm targeting), as in `git fetch upstream master && git rebase -i upstream/master`. That's much easier than having to work out how many steps to jump back &c.